reference/technical_information/recommended_configuration.rst

```diff
@@ -17,19 +17,19 @@ Hardware Minimum Configuration
 Software
 --------
 
-+-------------------+--------------------------------------------------------------+
-| Debian (Linux)    | 7 (64 bits)                                                  |
-+-------------------+--------------------------------------------------------------+
-| or Ubuntu         | 14.04 (64 bits)                                              |
-+-------------------+--------------------------------------------------------------+
-| Apache Web Server | ≥ 2.4                                                        |
-+-------------------+--------------------------------------------------------------+
-| PHP (CGI not FPM) | ≥ 5.5                                                        |
-+-------------------+--------------------------------------------------------------+
-| MySQL             | ≥ 5.1 (sufficient for most projects, depending on volumetry) |
-+-------------------+--------------------------------------------------------------+
-| or MongoDB        | 2.4                                                          |
-+-------------------+--------------------------------------------------------------+
++-------------------+----------------------------------------------------------------------------+
+| Debian (Linux)    | 7 (64 bits)                                                                |
++-------------------+----------------------------------------------------------------------------+
+| or Ubuntu         | 14.04 (64 bits)                                                            |
++-------------------+----------------------------------------------------------------------------+
+| Apache Web Server | ≥ 2.4                                                                      |
++-------------------+----------------------------------------------------------------------------+
+| PHP (CGI not FPM) | ≥ 5.5                                                                      |
++-------------------+----------------------------------------------------------------------------+
+| MySQL             | 5.1 ≤ version ≤ 5.6 (sufficient for most projects, depending on volumetry) |
```

Comment:
I'd remove the part between parenthesis, I'd also remove the "or" word before MongoDB (because this is MySQL only Mysql AND MongoDB). You should add a note after 2.4 to say "See [system requirements](http://docs.akeneo.com/latest/developer_guide/installation/system_requirements/system_requirements.html) to determine wether you need it or not.